test/e2e/apps/disruption.go

```diff
@@ -317,7 +317,7 @@ var _ = SIGDescribe("DisruptionController", func() {
 
 			if c.shouldDeny {
 				err = cs.CoreV1().Pods(ns).EvictV1(ctx, e)
-				framework.ExpectError(err, "pod eviction should fail")
+				gomega.Expect(err).To(gomega.HaveOccurred(), "pod eviction should fail")
```

Comment:
Can this and the other `gomega.HaveOccurred()` be made more selective, i.e. use `gomega.MatchError`?

Reply:
sorry for late reply. will check it as soon as possible.

Reply:
has refined, please review . thanks.

Reply:
sorry, missed the other gomega.HaveOccurred(), fixed it. please review . thanks